recipes/dev/generate_v2_distributed.py

```diff
@@ -15,8 +15,9 @@
 
 from torchtune import config, training, utils
 from torchtune.data import load_image, Message, padded_collate_tiled_images_and_mask
-
 from torchtune.generation import sample
+from torchtune.models.llama3_2_vision import parallelize_llama3_2_vision
```

Comment:
A recipe shouldn't have model specific imports

Reply:
Bumping this comment, can we take this out?